apps/judicial-system/web/src/routes/Admin/Statistics/Statistics.tsx

```
@@ -0,0 +1,311 @@
+                                  text={formatMessage(
+                                    strings.allCasesStatisticsTooltip,
+                                  )}
+                                />
+                              </Text>
+                            ),
+                            values: [
+                              <LabelValue
+                                label="Heildarfjöldi"
+                                value={stats?.count}
+                              />,
+                            ],
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)